portfolio/src/main/webapp/script.js

```diff
@@ -45,8 +45,8 @@ function createListElement(text) {
 
 function displayComments() {
   fetch('/add-comment').then(response => response.json()).then((comments) => {
-      
       console.log(comments);
+      console.log("test:");
```

Comment:
nitpick: remove prints statements when submitting pull requests; only use it for tests